web/src/pages/RestaurantPage/RestaurantPage.js

```diff
@@ -0,0 +1,77 @@
+import { useQuery } from '@redwoodjs/web'
+import Template1 from 'src/components/Template1/Template1'
+import Template2 from 'src/components/Template2/Template2'
+
+const GET_RESTURANT_INFO = gql`
+  query GetInfo($id: String!) {
+    restaurantById(id: $id) {
+      id
+      brandColor
+      sheetId
+      theme
+      busInfo {
+        id
+        name
+        address {
+          addressNumber
+          addressStreet
+          postalCode
+          city
+          state
+          country
+        }
+        hours {
+          monday
+          tuesday
+          wednesday
+          thursday
+          friday
+          saturday
+          sunday
+        }
+        phone
+        description
+        valueProp1
+        valueProp2
+        valueProp3
+      }
+      menus {
+        id
+        name
+        items {
+          id
+          name
+          price
+          description
+        }
+      }
+    }
+  }
+`
+
+const templateOptions = { 1: Template1, 2: Template2 }
+
+const RestaurantPage = ({ id }) => {
+  const { loading, error, data } = useQuery(GET_RESTURANT_INFO, {
+    variables: { id },
+  })
+
+  if (loading) return 'Loading...'
```

Comment:
Not needed for this PR but it'd be cool if we had some nice loading states.

Reply:
@alicelovescake you can make a ticket so you don't forget about loading states. A simple way to do it would be using `react-icons` and then just animate it spinning with CSS. I believe Tailwind has an animation for spinning.